teehistorian/index.php

```diff
@@ -1,4 +1,50 @@
 <html>
+<?php if(isset($_SERVER['PHP_AUTH_USER'])) {
+	$auths = fopen("ddrace_auths.cfg", "r");
```

Comment:
Actually, should I put this in the public directory? Not sure if we want this public.